OracleDatabase/OracleRealApplicationClusters/README.md

```diff
@@ -0,0 +1,474 @@
+
+		#docker network create --driver=bridge --subnet=172.15.1.0/24 rac_pub1_nw
+
+		#docker network create --driver=bridge --subnet=192.168.17.0/24 rac_priv1_nw
+
+		Note: You must run  RAC on Docker on multi-host using MACVLAN Docker driver (see Docker documentation).
+   
+- If the docker bridge network is not available outside your host, you can use the Oracle Connection Manager (CMAN) image to access the RAC Database from outside the host. You need to download and setup Oracle Connection Manager image. For the setup, please refer README of Oracle Connection Manager image. 
+
+- RAC needs to run certain processes in real time mode. To run processes inside a container in real time mode, you need to make changes in your docker configuration files. Please update OPTIONS value in /etc/sysconfig/docker to following:
+
+	OPTIONS='--selinux-enabled --cpu-rt-runtime=950000 --cpu-rt-period=1000000         --exec-opt native.cgroupdriver=systemd'
+
+	Once you have editied the /etc/sysconfig/docker, execute following commands:
+
+		#systemctl daemon-reload
+
+		#systemctl stop docker
+
+		#systemctl start docker
+
+	For details, please refer
+    	https://docs.docker.com/engine/reference/commandline/dockerd/#examples
+
+- SELINUX must be in permissive mode.
+	
+- For Oracle RAC, certain kernel parameter need to be set. Please refer to Oracle Grid/DB install guide. Docker containers share certain kernel parameters with host. You need to set following parameters at host level in /etc/sysctl.conf:
+
+		fs.file-max = 6815744
+
+		net.core.rmem_max = 4194304
+
+		net.core.rmem_default = 262144
+
+		net.core.wmem_max = 1048576
+
+		net.core.wmem_default = 262144
+
+		net.core.rmem_default = 262144
+
+	Execute following once the file is modified.
+
+		#sysctl -a
+
+		#sysctl -p
+	
+- Verify you have enough memory and cpu resources available for container. Each container for RAC requires 8GB memory and 16GB swap.  For details, please refer to the install guide for Grid Infrastructure and database for your version.
+
+- The Oracle RAC docker file, does not contain any Oracle Software Binaries.  Download the following software from the Oracle Technology Network (OTN) and stage them under (DOCKER_RAC_IMAGE)/dockerfile/(VERSION) folder, where (DOCKER_RAC_IMAGE) refers to the location where you unzipped the Oracle RAC docker file.
+	- Oracle Database 12c Release 2 Grid Infrastructure (12.2.0.1.0) for Linux x86-64
+	- Oracle Database 12c Release 2 (12.2.0.1.0) Enterprise Edition for Linux x86-64
+
+- You must have downloaded the patch# p27383741_122010_Linux-x86-64.zip from OTN as per the instruction given under "FOLLOWING FREELY AVAILABLE PATCH IS REQUIRED FOR THIS IMAGE TO WORK" heading. Stage it under (DOCKER_RAC_IMAGE)/dockerfile/(VERSION) folder.
```

Comment:
This is the right place to mention the patch. You can remove the stuff at the top of the README as it's not necessary to be that loud and vocal about it. It's just another file that's required.